components/popover/popover-mixin.js

```diff
@@ -434,6 +517,123 @@ export const PopoverMixin = superclass => class extends superclass {
 		return 'block-end';
 	}
 
+	#getMobileTrayBlockStyleMaps() {
+
+		let maxHeightOverride;
+		const availableHeight = Math.min(window.innerHeight, window.screen.height);
+
+		// default maximum height for bottom tray (42px margin)
+		const mobileTrayMaxHeightDefault = availableHeight - minBackdropHeightMobile;
+		if (this._maxHeight) {
+			// if maxHeight provided is smaller, use the maxHeight
+			maxHeightOverride = Math.min(mobileTrayMaxHeightDefault, this._maxHeight);
+		} else {
+			maxHeightOverride = mobileTrayMaxHeightDefault;
+		}
+		maxHeightOverride = `${maxHeightOverride}px`;
+
+		const widthOverride = '100vw';
+
+		const widthStyle = {
+			minWidth: widthOverride,
+			width: widthOverride,
+			maxHeight: maxHeightOverride,
+		};
+
+		const contentWidthStyle = {
+			// set width of content in addition to width container so header and footer borders are full width
+			width: widthOverride
+		};
+
+		const contentStyle = {
+			...contentWidthStyle,
+			maxHeight: maxHeightOverride,
+		};
+
+		return {
+			'width' : widthStyle,
+			'content' : contentStyle,
```

Comment:
Nit (and same in the other similar method):
```suggestion
			width: widthStyle,
			content: contentStyle,
```